community/ubase/version

```diff
@@ -0,0 +1 @@
+1 1
```

Comment:
Alpine versions `ubase` using dates. Current is `20190312`. Though this is rather uncertain as Alpine is the only distributions shipping a non `-git` package for `ubase` on Repology.

https://repology.org/project/ubase/versions

Reply:
i suggest we make the version to be the date of the hash we checkout from mforney's repos for both sbase and ubase @dylanaraps

Reply:
I'm happy either way. :+1:

Reply:
thanks